backend/api/Areas/Admin/Mapping/Parcel/AddressMap.cs

```diff
@@ -0,0 +1,38 @@
+using Mapster;
+using System;
+using Model = Pims.Api.Areas.Admin.Models.Parcel;
+using Entity = Pims.Dal.Entities;
+
+namespace Pims.Api.Areas.Admin.Mapping.Parcel
+{
+    public class AddressMap : IRegister
+    {
+        public void Register(TypeAdapterConfig config)
+        {
+            config.NewConfig<Entity.Address, Model.AddressModel>()
+                .IgnoreNonMapped(true)
```

Comment:
I couldn't find a way to globally set this for Dependency Injection regrettably.  So every new configuration map should ignore non-mapped properties to ensure we don't introduce change between versions.

Reply:
Amazingly and intuitively Mapster would map nearly every property without individual configuration, but I think it makes more sense for maintenance over-time to individually configure them.